fltkhs.cabal

```diff
@@ -154,7 +187,12 @@ library
   if os(darwin)
      cc-options: -U__BLOCKS__ -D_Nonnull=
 
+----------------------------------------
+
 Executable fltkhs-fluidtohs
+  if !flag(executables)
+     buildable: False 
+
```

Comment:
`fltkhs-fluidtohs` needs to always be buildable. It's the preprocessor that allows you to drag-and-drop GUIs using the FLTK `Fluid` tool and generate Haskell code.

Reply:
ah, sorry about that, fluid must be the GUI builder you mentioned in the talk.